### DISCUSSION TITLE
Why doesn't `parseWithZod` Automatic type coercion of boolean types support ‘true’?

### DISCUSSION THREAD

**Discussion by shutootaki** [edmundhung/conform#793] [Q&A]
If you want to set a bool value for formData, you currently have to do the following.
This seems a bit redundant.
```
formData.append(‘isTimeUp’, String(isRequired && ‘on’));
```

Personally, I think that if the string ‘true’ is set, it could be converted to a boolean type in the same way as ‘on’.
Is there a reason to only support ‘on’?
Or is there already a simpler way to implement this?

**Comment by yoni-noma** [comment]
Also how can I pass 'false'?
